static/js/components/Candidate.jsx

```diff
@@ -32,6 +32,10 @@ export const useStyles = makeStyles(() => ({
   accordionItem: {
     maxWidth: "900px",
   },
+  position: {
+    fontWeight: "bold",
+    fontSize: "110%",
+  },
```

Comment:
```suggestion
```
This is actually not necessary as the `classes.position` class is actually imported from SourceMobile.jsx. Putting rules for `position` here will assign it to `candidateStyles.position` (candidate page specific styling not imported from the source page), but that's not necessary here.****